data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/stream/LogicalReplicationClient.java

```diff
@@ -131,4 +141,23 @@ private void closeStream() {
             }
         }
     }
+
+    private void categorizeError(Exception e) {
+        if (e.getMessage() != null && e.getMessage().contains(AUTHENTICATION_FAILED)) {
```

Comment:
same comment as above

Reply:
Added metrics for those specific errors.